src/main/java/mate/academy/dao/BookDao.java

```diff
@@ -0,0 +1,17 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.model.Book;
+
+public interface BookDao {
+    Book create(Book book);
+
+    List<Book> findAll();
+
+    Optional<Book> findById(Long id);
+
+    Book update(Book book);
+
+    boolean delete(Book book);
+}
```

Comment:
It seems that the task requires deleting a book by id, with the method taking an "Long id" parameter.